tests/unit/scripting.tcl

```diff
@@ -616,6 +616,56 @@ start_server {tags {"scripting"}} {
              [run_script {return redis.sha1hex('Pizza & Mandolino')} 0]
     } {da39a3ee5e6b4b0d3255bfef95601890afd80709 74822d82031af7493c20eefa13bd07ec4fada82f}
 
+    test "Measures elapsed time with redis.elapsed_start() and redis.elapsed_us()" {
+        # Test if redis.elapsed_us() correctly handles incorrect number of arguments and invalid timer object
+        assert_error {ERR* Wrong number of arguments for redis.elapsed_us*} {run_script {redis.elapsed_us()} 0}
+        assert_error {ERR* Invalid timer object*} {run_script {redis.elapsed_us("invalid")} 0}
+
+        # Test accurate calculation of Lua script execution time.
+        set escaped [run_script {
+            local start_timer = redis.elapsed_start()
+            while redis.elapsed_us(start_timer) < 1000000 do end
+            return {double = redis.elapsed_us(start_timer)}
+        } 0]
+        assert_morethan_equal $escaped 1000000 ;# 1 second (1 second = 1000000 microseconds)
```

Comment:
Both this and the next function are just testing that time is advancing, we should probably also test they are about a second long.

Reply:
Done with [`f696989` (#12971)](https://github.com/redis/redis/pull/12971/commits/f696989ece934dd15c401f1cafef2c54b6870c99)